Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_044.NpmCredentialsValidator.cs

```diff
@@ -101,7 +101,9 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NotFound ||
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NonAuthoritativeInformation)
                 {
-                    return ValidationState.NoMatch;
+                    message = $"The Uri {uri} is generic, no way to validate teh credential with for account {id}";
```

Comment:
>teh [](http://example.com/codeflow?start=77&length=3)

spelling here and in test file

Reply:
yes, corrected.

Reply:
1. bad grammar. 2. no use of apostrophes to enquote dynamic data. 3. incorrectly capitalized words. 

Use this:

$'{uri}' is a generic URI that can't be used to validate account '{account}'"

Please pay particular attention to this note, which is commonly given in code review. If you are confused about what I mean when I refer to 'dynamic data', we can follow-up in stand-up. It is always fine, of course, to ask for suggestions for user-facing content, which Hulon or I can provider (as I have here!). :)